src/test/Student.test.js

```diff
@@ -9,9 +9,22 @@ Enzyme.configure({ adapter: new Adapter() });
 
 describe('Student', () => {
   it('should render correctly', () => {
+<<<<<<< HEAD
     const studentData = {};
     const output = shallow(
       <Student studentData={studentData} />
+=======
+    const student = {
+      "_id":"5a0d089080bcabc5ac8cb222",
+      "name":"Frodo Baggins",
+      "picture":"https://www.w3schools.com/w3css/img_fjords.jpg",
+      "bio":"I am from someplace and I did something",
+      "__v":0,
+      "comments":[]
+    };
+    const output = shallow(
+      <Student studentData={student} key={student._id} />
+>>>>>>> Update test for Student component to include passed-in props.
```

Comment:
The arrows and equal signs are signs that you have an unresolved git merge issue

Reply:
Thanks for catching that. It's weird, I couldn't find these git merge artifacts in my local version. Anyhow, I re-pushed the file and the artifacts seem to be gone now.